src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/reports/ZipFilesSummaryReportItem.java

```diff
@@ -28,8 +28,11 @@
     @JsonProperty("container")
     public final String jurisdiction;
 
-    @JsonProperty("status")
-    public final String status;
+    @JsonProperty("last_event_status")
+    public final String lastEventStatus;
```

Comment:
Introducing envelope status created the need for this field to be more descriptive.